### DISCUSSION TITLE
How to get Text from LocalizedText?

### DISCUSSION THREAD

**Discussion by FrictionalUnemployment** [FreeOpcUa/python-opcua#1274] [Q&A]
So I'm trying to get the string value from key value "Text" 

`LocalizedText(Encoding:2, Locale:None, Text:Server)
LocalizedText(Encoding:0, Locale:None, Text:None)
LocalizedText(Encoding:2, Locale:None, Text:Sensors)
LocalizedText(Encoding:2, Locale:None, Text:Sensors)
LocalizedText(Encoding:2, Locale:None, Text:methods)
LocalizedText(Encoding:2, Locale:None, Text:methods)`

This is the method I'm using:
self.client.get_node(i).get_display_name()

Unsure if there is a method or if I have to just split it manually to get the value, but it seems to be a dictionary so shouldn't be as hard as I'm making it out to be.
How do I get only "methods" from "Text" key? so that the output becomes "methods"

**Comment by FrictionalUnemployment** [comment]
So I was able to solve it pretty quickly...
self.client.get_node(i).get_display_name().__dict__['_text']

Since self.client.get_node(i).get_display_name().__dict__ returns

{'Encoding': 2, '_text': 'Sensors', 'Locale': None, '_freeze': True}

I was under the impression that the key value was 'Text'. Weird.

**Reply by AndreasHeine** [reply]
https://github.com/FreeOpcUa/python-opcua/blob/0d595e596c9922699f0b36abb79583d5763eeb9b/opcua/ua/uatypes.py#L501

self.client.get_node(i).get_display_name()._text !?
